functions/portal/src/supabase-storage/supabase.storage.client.ts

```diff
@@ -0,0 +1,277 @@
+import { BadRequestException, InternalServerErrorException } from "@danet/core";
+import { contentType } from "npm:mime-types@2.1.35";
```

Comment:
NITPICK: add package version to `import.json`, also `mime` can be removed in `import.json`